test/lbp.jl

```diff
@@ -245,18 +245,22 @@ facts("Local Binary Patterns") do
 		@fact multi_block_lbp(img, 1, 1, 4, 4) --> 18
 	end
 
-	context("Descriptor") do
+	#=context("Descriptor") do
```

Comment:
To make them robust, it can be replaced by a binary image whose descriptor is manually calculated. 

@annimesh2809 If you would like to do this, let me know. If not, I'll do this one 😄